### PR TITLE
[DO NOT MERGE] various OCR soak fixes

### PR DESCRIPTION
**Pull request by Tofel**
* use EIP-1559 for sending funds (if enabled)
* validate Seth config
* return early on error during funds sending
* add default gas limit for Mumbai (6M)
* got rid of `deepcopy` library and instead copy `TestConfig` by marshalling and umarshalling (deep copy wasn't copying our custom struct that represents Duration -- probably due to it having a private field)

EDIT: impossible to unmarshall to an interface and sometimes we do that, need to think what do about it

### CODE REVIEW

File: integration-tests/utils/seth.go [hunk 1]
```diff
@@ -82,3 +83,46 @@ func MustReplaceSimulatedNetworkUrlWithK8(l zerolog.Logger, network blockchain.E
 
 	return network
 }
+
+// ValidateSethNetworkConfig validates the Seth network config
+func ValidateSethNetworkConfig(cfg *seth.Network) error {
```
Comment (AnieeG) [inline integration-tests/utils/seth.go:88]: nil check for cfg?

**Review by AnieeG — DISMISSED** [review]

File: integration-tests/utils/seth.go (hunk 1 above)
Comment (Tofel) [reply integration-tests/utils/seth.go:88]: 🤦 very good idea!

**Review by tateexon — DISMISSED** [review]

File: integration-tests/testconfig/testconfig.go [hunk 2]
```diff
@@ -157,13 +156,19 @@ func (c *TestConfig) Save() (string, error) {
 }
 
 // MustCopy Returns a deep copy of the Test Config or panics on error
-func (c TestConfig) MustCopy() any {
-	return deepcopy.MustAnything(c).(TestConfig)
-}
+func (c *TestConfig) MustCopy() TestConfig {
```
Comment (AnieeG) [inline integration-tests/testconfig/testconfig.go:159]: Does it make sense to have it as util method in CTF? which will take any interface return the copy?

Comment (Tofel) [reply integration-tests/testconfig/testconfig.go:159]: it does :-)

**Review by AnieeG — APPROVED** [review]

**Pull request closed without merging**